cli_reference/openshift_developer_cli/managing-environment-variables-in-odo.adoc

```diff
@@ -0,0 +1,10 @@
+[id='managing-environment-variables']
+= Managing envirnoment variables
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: managing-envirnoment-variables
+toc::[]
+
+{odo-title} stores component-specific configurations and environment variables in the `config` file. {odo-short} provides the `odo config` command to set, unset, and list environment variables for components without the need to modify the `config` file.
```

Comment:
Suggest saying "You can use the `odo config` command to set..." instead of "odo provides".